### PR TITLE
feat: swap ui-extension-sdk to app-sdk

### PR DESCRIPTION
**Pull request by Jwhiles**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1451,6 +1451,11 @@
   resolved "https://registry.yarnpkg.com/@commitlint/types/-/types-11.0.0.tgz#719cf05fcc1abb6533610a2e0f5dd1e61eac14fe"
   integrity sha512-VoNqai1vR5anRF5Tuh/+SWDFk7xi7oMwHrHrbm1BprYXjB2RJsWLhUrStMssDxEl5lW/z3EUdg8RvH/IUBccSQ==
 
+"@contentful/app-sdk@^3.29.2":
```
Comment (suevalov) [inline yarn.lock:1454]: Looks like `ui-extensions-sdk` is used somewhere, otherwise I'd expect it to be removed from `yarn.lock`

Comment (suevalov) [reply yarn.lock:1454]: Never mind, just saw your push

**Review by suevalov — APPROVED** [review]
🚢

**Comment by suevalov** [comment]
Closes https://github.com/contentful/field-editors/issues/575

**Review by andipaetzold — APPROVED** [review]

**Pull request merged**